plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/GroupMetadataServiceImpl.java

```diff
@@ -42,11 +48,29 @@
 {
   private final NpmGroupRepository npmGroupRepository;
 
+  private final AlphanumComparator comparator;
+
+  private boolean mergeMetadata = SystemPropertiesHelper.getBoolean(
+      "com.bolyuba.nexus.plugin.npm.service.internal.GroupMetadataServiceImpl.mergeMetadata",
```

Comment:
true it is more refactor friendly, but this is not consistency friendly.

Didn't we already discuss that using feature specific system properties was more appropriate than class names? 

https://github.com/sonatype/nexus-oss/commit/0f49ea06e49294817077f6bffeb32138f0a5cc2f

It is more helpful and future proof to have system properties that do not depend on class names. We have less to worry about in refactor - in fact, my understanding is the new npm plugin will use sonatype packages, and so will likely change this property name unless we set it to something more generic.

Something like: nexus.npm.mergeGroupMetadata

I would also be OK with not even having a system property unless the use case for having it is strong.